recipe/meta.yaml

```diff
@@ -47,14 +61,15 @@ test:
     - papermill --help
 
 about:
-  home: https://github.com/nteract/papermill
-  license_file: LICENSE
+  home: http://github.com/nteract/papermill
   license: BSD 3-Clause
   license_family: BSD
-  summary: 'Papermill is a tool for parameterizing, executing, and analyzing Jupyter Notebooks.'
-  doc_url: http://papermill.readthedocs.io/en/latest/
+  license_file: LICENSE
+  summary: "Papermill parameterizes, executes, and analyzes Jupyter Notebooks."
+  doc_url: http://papermill.readthedocs.io/
   dev_url: https://github.com/nteract/papermill
 
 extra:
   recipe-maintainers:
-    - pmlandwehr
```

Comment:
@pmlandwehr had been inactive lately so let's merge this to get this project updated.
We can always re-add @pmlandwehr if he wants to maintain the feedstock again.

Reply:
Thanks @ocefpaf :D

Reply:
Thank you for helping to maintain this feedstock. And thnaks @isuruf for the review.

Reply:
Great! Yes, I've clocked out of a lot, and am grateful for anyone interested in helping to maintain a feedstock!